docs/docs/build/rill-project-file.md

```diff
@@ -1,32 +1,161 @@
-<br />
+
+## Rill Project Files
+
+When you create a new Rill project, the following files are automatically generated:
+
+- **`rill.yaml`** - Central configuration hub for your entire project
+- **`connectors/<connector>.yaml`** - Connector configuration files (e.g., `duckdb.yaml`, `clickhouse.yaml`)
+- **`.env`** - Environment variables and sensitive configuration data
+- **`.gitignore`** - Git ignore rules for the project
+
+### `rill.yaml`
+The `rill.yaml` file serves as the central configuration hub for your entire project. While often overlooked, this powerful file enables you to set project-wide defaults, configure environment variables, define connector settings, create test users, and establish security policies across all your metrics views and dashboards.
+
+Let's walk through the key capabilities of `rill.yaml`, including setting up [`mock_users`](/build/metrics-view/security#testing-policies-in-rill-developer) to test row access policies, configuring default [security settings](#metrics-views-security-policy) for your metrics views and explore dashboards, establishing [refresh schedules](#model-refresh-schedule), defining MCP [`ai_instructions`](#ai_instructions), setting the [default OLAP connector](#olap-connector) and more.
 
 For a list of all supported settings, see our [project YAML reference page](/reference/project-files/rill-yaml).
 
+### `.env`
+
+The `.env` file stores environment variables and sensitive configuration data for your Rill project. This includes database connection strings, API keys, and other credentials that should not be committed to version control. Use this file to securely manage [connection credentials](/connect/credentials) and environment-specific settings. For more information about credential management, see our [credentials documentation](/connect/credentials).
+
+### `<connector>.yaml`
+
+The `<connector>.yaml` file in the `connectors/` directory depends on which OLAP engine you initialized your project with (DuckDB, ClickHouse, Druid, or Pinot) and serves as the default OLAP engine for your project. When starting a blank project, this always defaults to `duckdb.yaml`.
+
+### `.gitignore`
+
+The `.gitignore` file specifies which files and directories should be ignored by Git version control. Rill automatically generates this file with appropriate rules to exclude sensitive files like `.env`, temporary files, and build artifacts from being committed to your repository.
+
+
+## OLAP Connector
+When you add an OLAP connector to your project, Rill automatically updates the `olap_connector` field in `rill.yaml` with the new connector (e.g., `ClickHouse`, `Druid`). If you create multiple connectors, Rill will append "_#" to the file name and use this as the default connector.
+
+```yaml
+olap_connector: clickhouse
+```
+
+The default OLAP connector is used as the default `output` for all of your models unless otherwise specified.
 
-## Model Defaults
+## Project Defaults
```

Comment:
Hmm. Providing all the defaults here in this first "Build" page seems out-of-order. At this point, we haven't yet introduced models, metrics views, dashboards, etc.